packages/exo/test/test-label-instances.js

```diff
@@ -0,0 +1,117 @@
+// eslint-disable-next-line import/order
+import { test } from './prepare-test-env-ava-label-instances.js';
+import { passStyleOf } from '@endo/far';
+import { M } from '@endo/patterns';
+import {
+  defineExoClass,
+  defineExoClassKit,
+  makeExo,
+} from '../src/exo-makers.js';
+
+const { quote: q } = assert;
+
+const UpCounterI = M.interface('UpCounter', {
+  incr: M.call()
+    // TODO M.number() should not be needed to get a better error message
+    .optional(M.and(M.number(), M.gte(0)))
+    .returns(M.number()),
```

Comment:
type signature not relevant to this test.

consider simplifying 
```suggestion
  incr: M.call().returns(undefined),
```

or simply omitting the interface guards

Reply:
Got rid of the optional arg as suggested, but kept the return, so the incremented value would still be observable. That observation isn't actually relevant, but without it the code is confusingly vacuous.

Resolving.